examples/citrix_adc/README.md

```diff
@@ -0,0 +1,369 @@
+# Configuring secure application delivery using Citrix ADC and the _Venafi Provider for HashiCorp Terraform_
+
+In this example, we'll show you how to better secure application delivery using _Venafi Provider for HashiCorp Terraform_ with your Citrix ADC instance. Adding Venafi enables you to manage certificates more securely as part of the TLS termination process on your load balancer.
+<!-- 
+ORIGINAL TEXT: This example will guide you in mounting a [CITRIX-ADC](https://www.citrix.com/products/citrix-adc/) instance and make certificates for those sites using Venafi's product [HashiCorp Terraform](https://terraform.io/) implementation in order to provide [SSL termination](https://www.techwalla.com/articles/what-is-ssl-termination). -->
+<!--
+DW: Hi Luis, I think this first para should describe the desired outcome: state in simple terms what the user can expect when they implement your example. In fact, I re-wrote your title to state more directly what the desired outcome actually is (as I understand it): securing application delivery using Venafi Provider for HashiCorp Terraform with my Citrix ADC instance. So for example, while mounting a Citrix ADC instance is something the user will do to get to the outcome, it's not the end goal. And similarly, TLS termination is part of the process of securing app delivery using Citrix and Venafi, but not the desired end goal. Does that make sense? 
+-->
+
+## Who should use this example? <!-- Suggest not using "Persona" as this is an UX term used more internally in software dev than a term that users would typically understand; while more techy people will use your example and might understand UX notion of personas, they are in this context a user. -->
+
+The steps described in this example are typically performed by **DevOps engineers** or **system administrators**. Generally, you'll need a basic understanding of Citrix ADC, Venafi Trust Protection Platform or Venafi Cloud, and the required permissions for completing the tasks described in the example.
+
+<!--
+DW: So I suggest adding--as I tried to do in that second sentence--the basic knowledge (as well as the permissions and access to the various systems) that is required in order to successfully complete your example. 
```

Comment:
LGTM, Just a comment. In this Citrix scenario you should also have some basic knowledge of Bash command language in order to set your provider locally. Should this also be included here? Or should we give it for granted? 

This is something to consider that may happen in the future since not all Terraform providers may not be registered in Terraform Registry at that time (hence that's why we provided a "base installation for providers to be installed locally").

Reply:
Good point. I suggest you include it as a simple TIP-style note...something like: 

<**TIP** Having at least some basic knowledge of the Bash command language is helpful, such as when you need to set your provider locally.